google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpcViews.java

```diff
@@ -91,28 +93,49 @@ public class SpannerRpcViews {
    * measures the latency between Google's network receives an RPC and reads back the first byte of
    * the response. gfe_header_missing_count is a counter of the number of RPC responses without a
    * server-timing header.
+   *
+   * @deprecated The OpenCensus project is deprecated. Use OpenTelemetry to get gfe_latency and
+   *     gfe_header_missing_count metrics.
    */
   @VisibleForTesting
+  @ObsoleteApi(
+      "The OpenCensus project is deprecated. Use OpenTelemetry to get gfe_latency and gfe_header_missing_count metrics.")
   public static void registerGfeLatencyAndHeaderMissingCountViews() {
-    viewManager.registerView(SPANNER_GFE_LATENCY_VIEW);
-    viewManager.registerView(SPANNER_GFE_HEADER_MISSING_COUNT_VIEW);
+    if (SpannerOptions.isEnabledOpenCensusMetrics()) {
```

Comment:
Metrics its little different. OpenCensus Metrics and OpenTelmetry metrics can be enabled or disable separately. So enabling one will not interfere with other. Also customers don't have the option to disable OpenTelemetry metrics . 

If customer creates 1 client and enables metrics and creates another spanner client. Customer would only get metrics for the second client which is expected.

Reply:
Hmm.... I think it will depend on what is called in which order. But as this is OpenCensus, which only supports global configuration, I guess this is the best we can do.